provisioner/terraform/provision.go

```diff
@@ -350,3 +350,9 @@ func tryGettingCoderProviderStacktrace(sess *provisionersdk.Session) string {
 	}
 	return string(stacktraces)
 }
+
+// gitAuthAccessTokenEnvironmentVariable was copied from provider.GitAuthAccessTokenEnvironmentVariable
```

Comment:
I feel somewhat strongly that this change should be in its own PR.
If we find that this was the wrong approach for some reason, we won't be able to simply revert the PR without blowback.

Reply:
done. the separate PR was merged.